app/infrastructure/exceptions/db_exception.py

```
@@ -0,0 +1,16 @@
+class DatabaseException(Exception):
+    def __init__(self, message: str = "데이터베이스 오류가 발생했습니다.") -> None:
+        self.message = message
+        super().__init__(self.message)
+
+
+class DatabaseIntegrityException(DatabaseException):
+    def __init__(
+        self, message: str = "데이터베이스 무결성 오류가 발생했습니다."
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)